### ISSUE TITLE
Als developer wil ik kunnen tijdreizen in een component

### ISSUE THREAD

**Issue by matthiasoliveiro** [VNG-Realisatie/huwelijksplanner#183]
zodat ik een object kan terug halen op elk gegeven moment.

Dit is nodig voor Beheer en compliance.

**Comment by JanWillemKooi** [comment]
Is dit een user story die afgerond kan worden of is dit meer een soort definition of done ding of misschien iets voor een architectuur

**Comment by rubenvdlinde** [comment]
Ja, tijdreizen is een functionaliteit die kan worden toegevoegd. Dus daarna kan het issue worden gesloten.

**Comment by rubenvdlinde** [comment]
Kunnen we deze de volgende sprint in trekken? @CMasselink en @matthiasoliveiro ? Dan kan ik hem combineren met het bijwerken van de redocs en authorizatie. Wil niet elke keer alle componenten bijwerken...

Sizing daar alvast op ingezet.

**Comment by rubenvdlinde** [comment]
Deze komt mee met #274

**Comment by rubenvdlinde** [comment]
Dit is nu op de achtergrond geregeld, dus het zou mijn voorkeur hebben om komende sprint uit te schrijven hoe deze getest kan worden zodat we hem kunnen afronden.

**Comment by rubenvdlinde** [comment]
@rjzondervan voorbeeld test uitschrijven in postman en aanleveren bij @CMasselink zodat dit getest kan worden

**Comment by rjzondervan** [comment]
Eerst maar even een testbeschrijving gemaakt, een postmantest zou kunnen, maar heeft met de vaak wijzigende en ververste componenten zoals we die nu hebben wat risico's ;).

Het testen van de tijdreisfunctionaliteit kan als volgt:

Als we een component nemen waarin we weten dat resources vaak wijzigen, zoals het verzoekencomponent (waar ik dit zelf ook getest heb) op de development-omgeving.

Als we daar een request opvragen via https://vrc.dev.huwelijksplanner.online/requests/{request_id}/change_log krijgen we een lijst met de verschillende versies die deze resource gehad heeft. Neem dan een versie die stevig afwijkt van de versie die op het moment dat je deze test aan het uitvoeren bent geldig is. Kopieer vervolgens de property “dateModified” van deze versie.

Vervolgens kan er in deze resource tijdgereisd worden door de resource op te vragen met https://vrc.dev.huwelijksplanner.online/requests/{request_id}?validOn=YYYY-MM-DD%20HH:MM:SS met in de plaats van YYYY-MM-DD%20HH:MM:SS een datum/tijd tussen de dateModified van de versie die je hebt gekozen en de dateModified van de volgende versie, en controleer of de resource gelijk is met de gekozen versie.

**Comment by CMasselink** [comment]
Dit werkt
